src/Microsoft.Tye.Core/Serialization/ConfigServiceParser.cs

```diff
@@ -48,6 +47,14 @@ private static void HandleServiceNameMapping(YamlMappingNode yamlMappingNode, Co
                     case "dockerFile":
                         service.DockerFile = YamlParser.GetScalarValue(key, child.Value);
                         break;
+                    case "dockerFileArgs":
```

Comment:
@davidfowl thoughts on making these nested instead of flat? For example
```yaml
services
- name: foo
  docker:
    file: <path to file>
    context: <path to context directory>
    args: <args>

Reply:
Not as part of this change. That's a bigger one.